net/Socket.cpp

```diff
@@ -50,6 +50,7 @@ constexpr std::chrono::microseconds WebSocketHandler::PingFrequencyMicroS;
 
 std::atomic<bool> SocketPoll::InhibitThreadChecks(false);
 std::atomic<bool> Socket::InhibitThreadChecks(false);
+std::string Socket::SocketPath("/tmp/coolwsd.sock");
```

Comment:
Do we want this in a global variable really? surely we can store it on a UnixSocket on an instance or somesuch?  We already have randomized Unix sockets for the Prisoner connection:

    std::shared_ptr<ServerSocket> findPrisonerServerPort()

Could we clean this up for both uses somehow and have a UnixServerSocket sub-class that handles this with a virtual destructor in ServerSocket ? =)

Reply:
I did this because it was the fastest solution and I don't have so much time (I am not a C++ developer but after some tries I can write some nice code). I think someone more skilled could code this in few minutes/hours but I would probably loose days